index.html

```diff
@@ -412,6 +472,7 @@ <h1 class="About">About<br>Myself</h1>
   
     <button type="submit" class="gint" id="gint">Get in touch</button>
   </form>
+</div>
```

Comment:
- [ ] Same thing here. Kindly correct this too.

###  Your Design
![your-form-btn](https://user-images.githubusercontent.com/26597432/221515386-076425b9-2ff2-4a6d-a095-b918be928180.png)

###  Figma Design
![temp-form-btn](https://user-images.githubusercontent.com/26597432/221515429-47bd18fe-52a9-4e8b-93b6-9e13a424b9fb.png)

Reply:
adjusted the buttons as per requested.